docs/tools/metrics.md

```
@@ -0,0 +1,316 @@
+PCP uses a standardized unit system:
+
+| Unit | Description |
+|------|-------------|
+| `millisec` | Time (milliseconds) |
+| `Kbyte`, `Mbyte`, `Gbyte` | Data size |
+| `count` | Dimensionless count |
+| `none` | No units |
+
+### Example: Understanding a Metric
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)